assets/app/styles/_core.less

```diff
@@ -8,6 +8,7 @@ html, body {
 
 .console-os {
   background-color: @body-bg;
+  height: ~"calc(100% - 121px)"; // removes scrollbar on pages with less than full viewport content
```

Comment:
It's only calculated when main.css is generated, so it shouldn't have a performance issue.